src/platform/packages/shared/presentation/presentation_publishing/interfaces/fetch/publishes_pause_fetch.ts

```diff
@@ -11,10 +11,14 @@ import type { Observable } from 'rxjs';
 
 export interface PublishesPauseFetch {
   isFetchPaused$: Observable<boolean>;
+  setFetchPaused: (paused: boolean) => void;
```

Comment:
Can we separate this out into a different interface in the same file? IMO we shouldn't enforce that every API that pauses fetch can also have its fetch paused publicly.

Reply:
Can do 🫡

Reply:
Done in b5c75eea3d64e86bfd388ac485756bbd66a88cdd